workers/main/src/index.ts

```
@@ -0,0 +1,92 @@
+import * as path from 'path';
+import { readdirSync } from 'fs';
+import { Connection, ScheduleClient } from '@temporalio/client';
+
+const workflowsPath = path.join(__dirname, 'workflows');
+const activitiesPath = path.join(__dirname, 'activities');
+
+const activityModules = readdirSync(activitiesPath)
+  .filter((f: string) => (f.endsWith('.ts') || f.endsWith('.js')) && !f.endsWith('.test.ts') && !f.endsWith('.test.js'))
+  .map((f: string) => {
+    const mod = require(path.join(activitiesPath, f));
+    const validExports: Record<string, Function> = {};
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)